go.mod

```diff
@@ -2,6 +2,21 @@ module knative.dev/func
 
 go 1.20
 
+replace (
+	// Version used in Eventing
+	github.com/cloudevents/sdk-go/sql/v2 => github.com/cloudevents/sdk-go/sql/v2 v2.0.0-20220930150014-52b12276cc4a
+
+	// Use older version since it checks ~/.docker/config.json for credentials as opposed to later versions
+	github.com/containers/image/v5 => github.com/containers/image/v5 v5.23.1
+
+	// Tekton CLI uses newer version without currently used constructor code
+	github.com/hinshun/vt10x => github.com/hinshun/vt10x v0.0.0-20180809195222-d55458df857c
+	// Pin k8s.io dependencies to align with Knative and Tekton needs
+	k8s.io/api => k8s.io/api v0.25.4
+	k8s.io/apimachinery => k8s.io/apimachinery v0.25.4
+	k8s.io/client-go => k8s.io/client-go v0.25.4
```

Comment:
Depending on `Tekton` versions, the `k8s.io` could be at `0.26.x` to align with rest of Knative.

Reply:
Both my comments are OK to follow-up in single-purpose isolated PRs.